src/lib/configs.ts

```diff
@@ -20,7 +22,20 @@ export async function initializeConfigs(config: Config = {}): Promise<{ [key: st
   GLEE_DIR = path.resolve(GLEE_PROJECT_DIR, '.glee')
   GLEE_LIFECYCLE_DIR = path.resolve(GLEE_DIR, config.functionsDir || 'lifecycle')
   GLEE_FUNCTIONS_DIR = path.resolve(GLEE_DIR, config.functionsDir || 'functions')
-  GLEE_CONFIG_FILE_PATH = path.resolve(GLEE_DIR, 'glee.config.js')
+
+  GLEE_CONFIG_FILE_PATH_TS = path.resolve(GLEE_DIR, 'glee.config.ts')
+  GLEE_CONFIG_FILE_PATH_JS = path.resolve(GLEE_DIR, 'glee.config.js')
+  GLEE_CONFIG_FILE_PATH = existsSync(GLEE_CONFIG_FILE_PATH_TS) ? GLEE_CONFIG_FILE_PATH_TS : GLEE_CONFIG_FILE_PATH_JS
+
+  if(existsSync(GLEE_CONFIG_FILE_PATH_TS) && existsSync(GLEE_CONFIG_FILE_PATH_JS)) {
+    logWarningMessage(
+      `Both 'glee.config.js' and 'glee.config.ts' files were found at ${GLEE_DIR}. 
+      For IntelliSense support, 'glee.config.ts' will be used over 'glee.config.js'. 
```

Comment:
```suggestion
      The 'glee.config.ts' will be used and 'glee.config.js' will be ignored. 
```

I'd not mention IntelliSense as this is just one of the reasons TypeScript is preferred.